ang/civicase/contact/directives/contact-card.directive.js

```diff
@@ -97,6 +97,12 @@
        */
       function getInitials (contactFullName) {
         var names = contactFullName.split(' ');
+        var ifNameContainsHonorificStrings = _.contains(['Mr.', 'Ms.', 'Mrs.', 'Dr.'], names[0]);
```

Comment:
This would make Civicase very specific to english. In other languages we use other honorifics. Do you think it's possible to use the contact's first name and last name?

Reply:
Now I am fetching the prexies from the BE, that way both the translation and other honorifics are covered.

Reply:
Also I could not use First name and last name, because in some cases the name is sent from BE apis, and right now it just sends back the display name. So it will be a big architectural change to send an object instead of a string.